pyuvdata/uvcal/uvcal.py

```diff
@@ -3715,6 +4560,13 @@ def fast_concat(
             metadata agrees. Allowed values are: 'antenna', 'time', 'freq', 'spw',
             'jones' ('freq' is not allowed for delay or wideband objects and 'spw' is
             only allowed for wideband objects).
+        ignore_name : bool
+            Option to ignore the name of the phase center (`cat_name` in
+            `phase_center_catalog`) when combining two UVData objects. If set to True,
+            phase centers that are the same up to their name will be combined with the
+            name set to the name found in the first UVData object in the sum. If set to
+            False, phase centers that are the same up to the name will be kept as
```

Comment:
UVData -> UVCal

Reply:
Fixed in 766e32646768181b70ddd8d2e576d90ca57c254c.